test/expect-tests/dune_patch/dune_patch_tests.ml

```diff
@@ -147,3 +159,17 @@ let%expect_test "patching a deleted file" =
   [%expect {|
     File foo.ml not found |}]
 ;;
+
+let%expect_test "Using a patch from 'diff' with a timestamp" =
+  try
+    test [ "foo.ml", "This is wrong\n" ] ("foo.patch", unified);
+    check "foo.ml";
+    [%expect.unreachable]
+  with
+  | Dune_util.Report_error.Already_reported ->
+    String.drop [%expect.output] 53 |> print_endline;
```

Comment:
I don't think it is good to hardcode such a constant, given the temporary path locations aren't necessarily fixed length. Better use something like `String.lsplit2_exn` and `Filename.basename` to normalize the path.

Reply:
True, I just didn't want to spend too much time on this since it's going to be removed by the fix anyway, but having a green CI is always nice